config/deployments/control-plane-cass-operator-dev/kustomization.yaml

```diff
@@ -0,0 +1,11 @@
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
+
+namespace: k8ssandra-operator
+
+resources:
+  - ../control-plane
+
+images:
+  - name: k8ssandra/cass-operator
```

Comment:
While this might change the image to latest, it will not actually necessarily work, since the other changes from cass-operator master are not picked up with this approach. This includes RBAC changes, CRD changes etc

Reply:
Good points. It worked for my particular situation but not a robust solution. This needs some more work.

Reply:
@burmanm what do you think about changing the `kustomization.yaml` so that it points to the local cass-operator repo? Obviously people might have it stored in different places relative to their k8ssandra-operator repo, but they could work around that with a symlink.

Reply:
Yep, I guess ../cass-operator would be fine default.

Reply:
(add as many .. as necessary)

Reply:
I have been using this extensively to work with your rebuild branch. It makes things a lot easier.